docs/eventing/samples/kafka/README.md

```diff
@@ -56,33 +58,38 @@ If you want to run the Apache Kafka cluster on Kubernetes, the simplest option i
    $ kubectl apply -n kafka -f kafka.yaml
```

Comment:
They have to first download this file? Wonder if it would be better to have this use the here document? Or, maybe add a link to this file?
Also, the step says this will create a namespace, but it actually installs kafka?

Reply:
In the previous step which displays the yaml, i have added "Describe the size of your Apache Kafka installation in `kafka.yaml`", in the next step we apply this yaml.